### PR TITLE
Handle tp requests

### LINKED ISSUE #12: Let bots accept teleport requests

### PR DESCRIPTION
**Pull request by LowWeiLin**
Fixes #12 
Depends on #11 
Made changes to the definitions in flow-typed/mineflayer.js

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -21,8 +21,14 @@ console.log(process.argv);
 
 (require('mineflayer-navigate'))(mineflayer)(bot);
 
+bot.on('message', function(message) {
```
Comment (dariusf) [inline bot.js:24]: Maybe we could add this?

``` js
type Message = {
  extra: { text: string }[]
};

bot.on('message', function(message: Message) {
  // ...
});
```

So the type of the callback is known here, even though it's `Function` in the library definition. Wanted to put the type declaration there but apparently that's not supported yet.

**Review by dariusf — APPROVED** [review]

File: bot.js (hunk 1 above)
Comment (LowWeiLin) [reply bot.js:24]: Put the type declaration right above the function?

Comment (LowWeiLin) [reply bot.js:24]: I'm not sure if that is the only kind of object in it. Will check then do it.

Comment (dariusf) [reply bot.js:24]: Yeah, or at the top somewhere, up to you. I don't think we need to capture all possible types (it's never-ending...), just the ones we use.

**Pull request merged**